contracts/programs/bubblegum/src/error.rs

```diff
@@ -10,4 +10,18 @@ pub enum BubblegumError {
     HashingMismatch,
     #[msg("Unsupported Schema Version")]
     UnsupportedSchemaVersion,
-}
\ No newline at end of file
+    #[msg("Creator shares must sum to 100")]
+    CreatorShareTotalMustBe100,
```

Comment:
needed to add custom errors because MPL token metadata errors don't work with Anchor